src/utils/event.ts

```diff
@@ -57,7 +60,24 @@ export function warpEventCanonicalSignaturehash(eventName: string, argTypes: arg
   const funcSignature = `${eventName}(${argTypes.map(getArgStringRepresentation).join(',')})`;
   const funcSignatureHash = createKeccakHash('keccak256').update(funcSignature).digest('hex');
 
-  return `0x${(
-    BigInt(`0x${createKeccakHash('keccak256').update(funcSignatureHash).digest('hex')}`) & MASK_250
-  ).toString(16)}`;
+  const splitHash: bigint[] = toUintOrFelt(BigInt(`0x${funcSignatureHash}`), 256);
```

Comment:
You can use pattern matching, so you don't have to retrieve low and high by index

Reply:
> There is utility function to split into low and high please use that: Swapnil
I wanted to reuse the pre-defined function, so I used this function

Reply:
By pattern matching I meant the syntax `const [low, high] = foo()` so it would be more readable